cli/command/stack/remove.go

```diff
@@ -38,7 +38,10 @@ func runRemove(dockerCli command.Cli, opts removeOptions) error {
 	namespaces := opts.namespaces
 	client := dockerCli.Client()
 	ctx := context.Background()
-
+	version, err := client.ServerVersion(ctx)
```

Comment:
I think I tried adding `version` and `ClientVersion()` to `fakeClient{}`, but it didn't satisfy the Interface. Hm, let me check, perhaps I didn't